src/globals.test.js

```diff
@@ -1,8 +1,63 @@
 import * as globals from './globals';
 import { ConsoleWritesForTesting } from '../testUtils/consoleMocker';
-import { DEFAULT_MAX_SIZE_FOR_REQUEST } from './globals';
+import { DEFAULT_MAX_SIZE_FOR_REQUEST, Timer } from './globals';
 
 describe('globals', () => {
+  describe('Timer - tests', () => {
```

Comment:
Missing tests:
 - Taking value from env vars
 - "Stopping" instrumentation after time is passed
 - Timer is reseted every invocation